.gitignore

```diff
@@ -3,6 +3,7 @@
 /deps
 erl_crash.dump
 *.ez
+mix.lock
```

Comment:
Oh cripes! - I am so sorry. I really thought I had addressed this and all your other comments. I think GITting clumsiness has tripped me up again. :-(